src/components/Api.js

```diff
@@ -0,0 +1,197 @@
+export default class Api {
+  constructor({ baseUrl, headers }) {
+    this._baseUrl = baseUrl;
+    this._headers = headers;
+    this._token = headers.authorization;
+  }
+
+  getInitialCards() {
+    return fetch(`${this._baseUrl}/cards`, {
+      headers: {
```

Comment:
Needs correction: you should save the whole headers object in the constructor and reuse it in all the methods: `headers: this._headers`